src/inject/GPMInject/interface/customUI.js

```diff
@@ -21,7 +21,17 @@ const style = (elementSelector, styleObject) => {
   });
 };
 
+const cssRule = (styles) => {
+  const tag = document.createElement('style');
+  tag.type = 'text/css';
+  tag.appendChild(document.createTextNode(styles));
+  document.head.appendChild(tag);
+};
+
 window.wait(() => {
+  // Restore original text color on buttons in the Profile dialog (dark theme sets it to white).
+  cssRule('.gmusic-theme .material div.gb_ob a.gb_Ca.gb_nb {color: #666 !important}');
```

Comment:
Just going to point out here, that those classes you used won't work cross language/locale.  For some reason those classes change when going between different countries and using Google Play music.

Also this really should be PR'ed into https://github.com/gmusic-utils/gmusic-theme.js instead of patched here